api/v1alpha1/hosted_controlplane.go

```diff
@@ -96,6 +96,9 @@ type HostedControlPlaneStatus struct {
 	// +kubebuilder:validation:Optional
 	Version string `json:"version,omitempty"`
```

Comment:
why do we need both `version` and `ReleaseImage`?

Reply:
This is from our original API design... my understanding is users of HostedCluster want to know the semver for the release, so it has to bubble up from here as it's resolved at runtime. It's not used for reconciliation and is just additional metadata associated with the release image (again, just my understanding). cc @csrwng @sjenning @derekwaynecarr

Reply:
Fixed